coffee-tool/src/main/java/hu/icellmobilsoft/coffee/tool/common/FunctionalInterfaces.java

```diff
@@ -38,31 +42,32 @@ public FunctionalInterfaces() {
 
     /**
      * This and the following ones are for a smart wrapping of all action calls inside the REST implementations
-     * 
+     *
      * @param <R>
      *            return object type
      */
     @FunctionalInterface
-    public interface BaseExceptionSupplier<R> {
+    public interface BaseExceptionSupplier<R> extends hu.icellmobilsoft.coffee.se.function.BaseExceptionSupplier<R> {
         /**
          * Gets a result.
          *
          * @return a result
          * @throws BaseException
          *             exception
          */
+        @Override
         R get() throws BaseException;
```

Comment:
ezek a terhelt metodusok tulajdonkeppen nem is kellenenek, nem?

Reply:
A BaseRestService törik tőle, meg minden olyan hely, ahol felhívjuk az új funkcionális interfészeket. Szóval ha felülírjuk a fgv-eket a mostani módon, csak akkor marad kompatibilis a módosítás.